### ISSUE TITLE
Installing Mediawiki 1.30 created a window padding issue

### ISSUE THREAD

**Issue by philippenoel73** [Hutchy68/pivot#46]
### Setup

- Browsers and versions: Chrome Version 62.0.3202.94 (Official Build) (64-bit)
- MediaWiki version: 1.30
- Database version: MySQL 5.6.36-cll-lve
- PHP version: 5.6.32
- Pivot version: 1.0.5
- Extension version (if connected to the issue): None

### Issue
Hi,
I just installed Mediwiki 1.30 and the default window display is now reduced to 1024x768. See image attached. I searched but haven't found a parameter to allow the skin to be displayed full screen. Other skins are not creating that problem.

![pivot skin size padded](https://user-images.githubusercontent.com/34604074/34072218-45e8042e-e251-11e7-8975-e9e389261864.png)

**Comment by Hutchy68** [comment]
Thanks for reporting, I’ll investigate. It definitely should not be doing that.

**Comment by Hutchy68** [comment]
Could you try changing this in your pivot features array? Set `'IeEdgeCode' => 1,` to `'IeEdgeCode' => 0,` Although I suspect MW 1.30 is just adding the meta tag without testing if IE = 9 or since the skin tests for IE=9 there is a conflict with the shim they added. That meta tag **should NOT** be inserted into head of a browser that is not IE 9 and it is there for Chrome and Firefox. 😞

**Comment by Hutchy68** [comment]
Sorry for the delay, just had a chance to really look at this. Probably about time to have a real discussion on supporting IE 8. IE 8 support ended a while ago. Technically, IE 9 and 10 has ended too. Anyway, here is the fix.

Line 4 of `/assets/scripts/pivot.js` 
Change
```
if (!jQuery.support.cssFloat) { jQuery('html').addClass('lt-ie9').addClass('no-js'); }
```

to 

```
// if (!jQuery.support.cssFloat) { jQuery('html').addClass('lt-ie9').addClass('no-js'); }
```
I'm going to look at this further and probably just remove support for IE 8 completely. Or move it off to only load the CSS if IE 8 is detected properly. We have modernizr loaded too which would be a better way to detect.

**Comment by philippenoel73** [comment]
Thank you very much. Putting in comment line 4 of /assets/scripts/pivot.js make it work perfectly now.

Best Regards,
Phil

**Comment by BuildAndDie** [comment]
I was having this exact same problem, the real issue I had was that when viewing on mobile, it wasn't make the navigation sidebar go away. This fixed it all!

**Comment by kghbln** [comment]
Since MW is no longer supporting [IE 8](https://www.mediawiki.org/wiki/Compatibility#Browser_support_matrix) either this fix should imho go into 2.0 / Edit: Oops, I thought this was about Foreground. Still, pivot could ship with the fix as a 1.1.0?

**Comment by Hutchy68** [comment]
Closing, fixed in v2.0 found in develop branch
